chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,7 +150,8 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
-            d.fixed and not d.claimed?
+            next unless d.fixed || d.multipath?
```

Comment:
Seems fair enough to me, let me dig a bit deeper into this and see if:
 - I can simplify it
 - I can add a decent comment to it that its clear enough.